afs/media/js/views/components/workflows/project-collection-workflow/project-collection-final-step.js

```diff
@@ -13,6 +13,8 @@ define([
         this.collectionOfChildPhysThings = ko.observableArray();
 
         this.relatedResources.subscribe(function(val){
+            // 77d8cf19-ce9c-4e0a-bde1-9148d870e11c
+            const physicalThingType = '8ddfe3ab-b31d-11e9-aff0-a4d18cec433a';
```

Comment:
This should be renamed to at least physicalThingNodeId - since you're using it as the nodeId for tile data.